### PR TITLE
[ION-9594] Custom Log Appender Documentation Must Include Enforcements

### PR DESCRIPTION
**Pull request by dnishant**
[[ION-9594]](https://www.mulesoft.org/jira/browse/ION-9594) Added documentation to include required appenders.

### CODE REVIEW

File: modules/ROOT/pages/custom-log-appender.adoc [hunk 1]
```diff
@@ -44,6 +44,7 @@ You can use any Log4j appender. For information about configuring `log4j2.xml`,
 https://logging.apache.org/log4j/2.x/manual/configuration.html#ConfigurationSyntax[Log4j Configuration Syntax].
 
 To enable the Log4j appender, update the `log4j2.xml` configuration file with your logger settings.
+Include the following Log4j appenders `Log4J2CloudhubLogAppender` and `RollingFile` as shown in the example below along with your own logger settings.
```
Comment (hannanelson) [inline modules/ROOT/pages/custom-log-appender.adoc:47]:
```suggestion
Include the Log4j appenders `Log4J2CloudhubLogAppender` and `RollingFile` in the example along with your logger settings.
```

Comment (hannanelson) [inline modules/ROOT/pages/custom-log-appender.adoc]: Can we change these lines like this:

To enable the Log4j appender, update the `log4j2.xml` configuration file with your logger settings and include the Log4j appenders `Log4J2CloudhubLogAppender` and `RollingFile`.
In Anypoint Studio, the `log4j2.xml` file is located in the `src/main/resources` directory.

File: modules/ROOT/pages/custom-log-appender.adoc [hunk 2]
```diff
@@ -53,6 +54,9 @@ Sends log data to CloudHub.
 RollingFile::
 Sends log data to the filesystem of the VM.
 
+[WARNING]
+If the appenders mentioned above are not part of the `log4j2.xml` file, Cloudhub support may lose visibility into the logs. It is strongly advised to include the above appenders in your custom configurations.
```
Comment (hannanelson) [inline modules/ROOT/pages/custom-log-appender.adoc]:
```suggestion
You must include the appenders in your custom configurations in the `log4j2.xml` file to enable MuleSoft Support team visibility into the logs.
```

**Comment by dnishant** [comment]
@hannanelson I've made the changes you suggested

**Review by MichelleZMY — APPROVED** [review]
LGTM

**Comment by dnishant** [comment]
Will need to re-open the PR with the correct git user for cla to pass, Will link this in the new PR.

**Pull request closed without merging**